apps/annotation/admin.py

```diff
@@ -5,12 +5,47 @@
 
 from apps.annotation.models import Annotation
 from apps.annotation.models import AnnotationRequest
-
+from urllib.parse import urlparse
 admin.site.register(get_user_model(), UserAdmin)
 
 
+def truncate(string, allowed_max=20):
```

Comment:
Here is BTW the use of django util and https://github.com/PrzypisPowszechny/pp/blob/493d4d4882ef03ed2d5a15e2119ead83354607af/apps/publisher/demagog.py#L116